_includes/pbc/all/install-features/202407.0/install-the-product-management-ai-feature.md

```diff
@@ -0,0 +1,254 @@
+
+<!--When "composer require" includes a "Spryker Feature" (github.com/spryker-feature), then it always needs to be "dev-master" version. For regular modules, use regular versions.-->
+
+```bash
+composer require spryker-eco/product-management-ai:"^0.1.1" --update-with-dependencies
+```
+
+**Verification**
+
+Make sure the following modules have been installed:
+
+| MODULE              | EXPECTED DIRECTORY <!--for public Demo Shops--> |
+|---------------------|-------------------------------------------------|
+| OpenAi              | vendor/spryker-eco/open-ai                      |
+| ProductManagementAi | vendor/spryker-eco/product-management-ai        |
+
+---
+
+### Set up database schema
+<!--If the feature has transfer object definition changes, merge the steps as described in [Set up database schema and transfer objects](#set-up-database-schema-and-transfer-objects). Provide code snippets with DB schema changes, describing the changes before each code snippet. Provide the console commands to apply the changes in project and core. -->
+
+Set up database schema as follows:
+
+1. Adjust the schema definition so entity changes trigger events:
+
+| AFFECTED ENTITY | TRIGGERED EVENTS |
+|-----------------|------------------|
+| SpyProductImage | *                |
+
+```xml
+<?xml version="1.0"?>
+<database xmlns="spryker:schema-01" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="zed" xsi:schemaLocation="spryker:schema-01 https://static.spryker.com/schema-01.xsd" namespace="Orm\Zed\ProductImage\Persistence" package="src.Orm.Zed.ProductImage.Persistence">
+
+    <table name="spy_product_image">
+        <column name="alt_text" type="VARCHAR" size="255" required="false"/>
+    </table>
+
+</database>
+```
+
+2. Apply database changes and generate entity and transfer changes:
+
+```bash
+console transfer:generate
+console propel:install
+console transfer:generate
+```
+
+**Verification**
+<!--Describe how a developer can check they have completed the step correctly.-->
+
+<!--Each step needs verification to make sure that the customer did not skip anything unintentionally.
+The verification needs to cover the entire "step".
+The verification step often needs to use an example domain, use
+ - "mysprykershop.com"
+ - "zed.mysprykershop.com"
+ - "glue.mysprykershop.com"
+domains according to your requirements.-->
+
+Make sure that following changes have been applied by checking your database:
+
+| DATABASE ENTITY   | TYPE     | EVENT |
+|-------------------|----------|-------|
+| spy_product_image | alt_text | *     |
+
+---
+
+### Set up transfer objects
+<!--Provide the following with a description before each item:
+* Code snippets with DB schema changes.
+* Code snippets with transfer schema changes.
+* The console command to apply the changes in project and core. -->
+
+Set up transfer objects as follows:
+
+```bash
+console transfer:generate
```

Comment:
transfers have been generated already, DB and transfer changes must be in 1 section if both exist